judge/forms.py

```diff
@@ -284,3 +285,20 @@ def clean_key(self):
         if Contest.objects.filter(key=key).exists():
             raise ValidationError(_('Contest with key already exists.'))
         return key
+
+
+class ProblemPointsVoteForm(ModelForm):
+    class Meta:
+        model = ProblemPointsVote
+        fields = ['points', 'note']
+        # The default 'required' error message is triggered if non-integer data is sent,
+        # even if the user enters data into the field. It is overridden here to provide a
+        # suitable response, along with the other errors for consistency.
```

Comment:
You should mention why an empty value is sent (because of how `.serialize` works in Javascript.